### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by GoldNotch**

### CODE REVIEW

File: client/src/scivi-editor.js [hunk 1]
```diff
@@ -814,68 +820,81 @@ SciViEditor.prototype.startComm = function (address, addressCorrespondences, eon
...
-                            if (!isInput) {
-                                var dfdNodeID = cor[j][0];
-                                var socketNmb = cor[j][2];
-                                var dfdNode = _this.getNodeByID(dfdNodeID);
-                                if (!dfdNode.data.outputDataPool)
-                                    dfdNode.data.outputDataPool = [];
-                                for (var k = dfdNode.data.outputDataPool.length; k <= socketNmb; ++k)
-                                    dfdNode.data.outputDataPool.push(null);
-                                dfdNode.data.outputDataPool[socketNmb] = msg[i][key];
+            Object.keys(msg)
```
Comment (icosaeder) [inline client/src/scivi-editor.js:865]: msg here is just a dict, not an array of dicts? The previous code was
for (var i = 0, n = msg.length; i < n; ++i) {
      Object.keys(msg[i]).forEach(function (key) {

File: client/src/scivi-editor.js [hunk 2]
```diff
@@ -814,68 +820,81 @@ SciViEditor.prototype.startComm = function (address, addressCorrespondences, eon
...
+                            last_outputs[socketNmb] = msg[key];
                         }
                     }
-                });
-            }
-            _this.process();
+                }
+            });
+
+            console.log('message received');
```
Comment (icosaeder) [inline client/src/scivi-editor.js:896]: Please remove the log.

File: kb/csv/cormap.ont [hunk 3]
```diff
@@ -1,4 +1,30 @@
 {
+    "history": [
```
Comment (icosaeder) [inline kb/csv/cormap.ont:2]: Please revert this file entirely.

File: lib/eye/csvRecorder.js [hunk 4]
```diff
@@ -47,6 +47,7 @@ if (IN_VISUALIZATION) {
 
         ADD_VISUAL(container);
     }
+    //console.log(INPUT["Table"]);
```
Comment (icosaeder) [inline lib/eye/csvRecorder.js:50]: Please revert this file.

File: lib/eye/eyeview.js [hunk 5]
```diff
@@ -196,6 +196,7 @@ if (IN_VISUALIZATION) {
             highlightAOI([ gaze[1] * imgSize[0], gaze[2] * imgSize[1] ], [ 100, 100 ]);
         if (gaze[gaze.length - 1] === "SELECT")
             toggleAOI(gaze[gaze.length - 2]);
+
```
Comment (icosaeder) [inline lib/eye/eyeview.js:199]: Please revert this file

File: lib/eye/vrboard.js [hunk 6]
```diff
@@ -64,15 +64,20 @@ if (IN_VISUALIZATION)
...
     }
+    if (CACHE["WallLog"])
+    {
+        OUTPUT["Wall Log"] = CACHE["WallLog"];
+        CACHE["WallLog"] = null;
+    }
         
     
-    if (CACHE["webSocket"]) 
+    if (CACHE["webSocket"] )
```
Comment (icosaeder) [inline lib/eye/vrboard.js:79]: Excessive space :)

File: client/src/scivi-editor.js (hunk 1 above)
Comment (GoldNotch) [reply client/src/scivi-editor.js:865]: Yes, it's a dict, because DataWebSocket in server sends dict {Addr: msg}

**Pull request merged**